TUnit.Assertions/AssertConditions/BecauseReason.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Globalization;
+
+namespace TUnit.Assertions.AssertConditions;
+
+internal class BecauseReason(string because, object[] becauseArgs)
+{
+    private string? _message;
+
+    private string CreateMessage()
+    {
+        const string prefix = "because";
+        try
+        {
+            string message = because.Trim();
+
+            if (becauseArgs?.Length > 0)
+            {
+                message = string.Format(CultureInfo.InvariantCulture, message, becauseArgs);

```

Comment:
I feel like string.Format is a relic of the past. Users can just pass in their own $"Interpolated {strings}". So I'd probably remove the `becauseArgs` to simplify things.

Reply:
You are probably right 👍